File: codeforces/nextRound.py
n, k = map(int, input().split())
scores = list(map(int, input().split()))

nextRound = 0
for i in range(len(scores)):
    if scores[i] >= scores[k - 1] and scores[i] > 0:
        nextRound += 1

print(nextRound)

File: codeforces/nextRound.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){

    int n, k;

    cin >> n >> k;

    int scores[n];

    for(int i = 0; i < n; i++){
        cin >> scores[i];
    }

 
    int nextRound = 0;
    for(int i = 0; i < n; i++){
        if(scores[i] >= scores[k - 1] && scores[i] > 0){
            nextRound++;
        }
    }

    cout << nextRound << endl;

    return 0;
}